src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/Batch/BatchTests.cs

```diff
@@ -55,21 +55,19 @@ public static void ConnectionCanCreateBatch()
         [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup))]
         public static void SqlBatchCanCreateParameter()
         {
-            using (var connection = new SqlConnection(DataTestUtility.TCPConnectionString))
-            using (var batch = connection.CreateBatch())
-            {
-                SqlBatchCommand batchCommand = new SqlBatchCommand("SELECT @p");
-
-                Assert.True(batchCommand.CanCreateParameter);
-                SqlParameter parameter = batchCommand.CreateParameter();
-                Assert.NotNull(parameter);
-                parameter.ParameterName = "@p";
-                parameter.Value = 1;
-                batchCommand.Parameters.Add(parameter);
-
-                batch.ExecuteNonQuery();
-
-            }
+            using var connection = new SqlConnection(DataTestUtility.TCPConnectionString);
+            connection.Open();
```

Comment:
I just addressed the failures on net8 and did not check the rest. 
Basically in my opinion this fix should have had a separate PR to address the test fix, but since it just showed up with net8 changes I included the fix in this PR.

Reply:
> Just a comment here: In general, it appears this class uses curly braces { ... } for large code blocks, with the exception of nested usings. I'm indifferent on whether you change this or not.

@H-Yeo 
There is a lot of existing code with that style. When C# 8 eliminated the braces requirement, we started allowing it in code and using it more heavily in test code. There's no need to keep new code consistent with old code in this respect and contributors are free to convert old style to new as they see fit. In non-test code, we are more thoughtful about the using scope so we tend to keep the braces for clarity, but they aren't strictly required.